rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -86,41 +86,46 @@ $.fn.dataTableExt.afnSortData['changes'] = function ( oSettings, iColumn )
 
 
 
-function computeChangeGraph(changes, id, currentRowsIds, changeCount) {
+function computeChangeGraph(changes, id, currentRowsIds, changeCount, displayGraph) {
   recentChanges[id] = changes;
   recentChangesCount[id] = changeCount
   if (currentRowsIds.indexOf(id) != -1) {
-    generateRecentGraph(id)
+    generateRecentGraph(id, displayGraph)
   }
 }
 
-function generateRecentGraph(id) {
-  var changes = recentChanges[id]
-  if (changes !== undefined) {
-    var graphId = "Changes-"+id
-    var data = changes.y
-    data.splice(0,0,'Recent changes')
-    var x = changes.x
-    x.splice(0,0,'x')
-    var chart = c3.generate({
-        size: { height: 30 , width: 168 }
-      , legend: { show: false }
-      , data: {
-            x: 'x'
-          , columns: [ x, data ]
-          , type: 'area-step'
-        }
-      , axis: {
-            x: {
-                show : false
-              , type: 'categories'
-            }
-          , y: { show : false }
-        }
-    });
-    recentGraphs[id] = chart
+function generateRecentGraph(id, displayGraph) {
+  var graphId = "Changes-"+id
+  if (displayGraph) {
+    var changes = recentChanges[id]
+    if (changes !== undefined) {
+      var data = changes.y
+      data.splice(0,0,'Recent changes')
+      var x = changes.x
+      x.splice(0,0,'x')
+      var chart = c3.generate({
+          size: { height: 30 , width: 168 }
+        , legend: { show: false }
+        , data: {
+              x: 'x'
+            , columns: [ x, data ]
+            , type: 'area-step'
+          }
+        , axis: {
+              x: {
+                  show : false
+                , type: 'categories'
+              }
+            , y: { show : false }
+          }
+      });
+      recentGraphs[id] = chart
```

Comment:
most lines don't finish by ; , but certain do
could you be consistent and finish them all by ; ?